ips.go

```diff
@@ -0,0 +1,145 @@
+package utils
+
+import "net"
+
+// IsIPv4 is plagiarism of net.ParseIP,
+// but without check for IPv6 case and without returning net.IP slice, whereby IsIPv4 makes no allocations.
```

Comment:
"plagiarism" feels like too strong of a word, maybe reword the comment to say

```go
// IsIPv4 is a copy of the standard net.ParseIP function but without special checks
// for IPv6 and without returning a net.IP slice, thus resulting in no allocations.
```
same for `IsIPv6`